src/sentry/static/sentry/app/views/releaseDetails.jsx

```diff
@@ -18,6 +21,54 @@ import TimeSince from 'app/components/timeSince';
 import Version from 'app/components/version';
 import withEnvironmentInQueryString from 'app/utils/withEnvironmentInQueryString';
 
+class ReleaseDetailsActions extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    release: PropTypes.object.isRequired,
+  };
+
+  handleDelete = () => {
+    let {orgId, projectId, release, api} = this.props;
+    let version = encodeURIComponent(release.version);
+    let path = `/organizations/${orgId}/releases/${version}/`;
+    let redirectPath = `/${orgId}/${projectId}/releases/`;
+    let indicator = IndicatorStore.add(t('Deleting Release...'));
```

Comment:
We've started to move to use the relevant action creators in `app/actionCreators/indicators` instead of directly accessing the stores.

Reply:
Ok. I'll update to use those functions instead.